.changeset/breezy-pumas-own.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-graph': major
+'@finos/legend-application-studio': minor
+'@finos/legend-query-builder': minor
+---
```

Comment:
Can you please split the changeset. Just add an empty changeset for `legend-query-builder` with patch change